From f3f3e88763632c62c4d554ee49124d075d78423b Mon Sep 17 00:00:00 2001
From: Wang Quanyang <quanyang.wang@windriver.com>
Date: Fri, 12 Jul 2019 05:25:57 -0400
Subject: [PATCH] gpu: mail-400: use new timer API ktime_get_boottime

Signed-off-by: Quanyang Wang <quanyang.wang@windriver.com>
---
 linux/mali_osk_time.c | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/linux/mali_osk_time.c b/linux/mali_osk_time.c
index 3f2a9d9..a4e389f 100644
--- a/linux/mali_osk_time.c
+++ b/linux/mali_osk_time.c
@@ -53,7 +53,5 @@ u64 _mali_osk_time_get_ns(void)
 
 u64 _mali_osk_boot_time_get_ns(void)
 {
-	struct timespec tsval;
-	get_monotonic_boottime(&tsval);
-	return (u64)timespec_to_ns(&tsval);
+	return ktime_get_boottime();
 }
-- 
2.22.0

